<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>preferencesDialog</class>
 <widget class="QDialog" name="preferencesDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>540</width>
    <height>307</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Preferences</string>
  </property>
  <widget class="QDialogButtonBox" name="buttonBox">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>271</y>
     <width>521</width>
     <height>31</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
   <property name="standardButtons">
    <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
   </property>
   <property name="centerButtons">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QTabWidget" name="tabWidget">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>521</width>
     <height>261</height>
    </rect>
   </property>
   <property name="currentIndex">
    <number>0</number>
   </property>
   <widget class="QWidget" name="inputTab">
    <attribute name="title">
     <string>&amp;General</string>
    </attribute>
    <widget class="QLabel" name="label">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>20</y>
       <width>111</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>&amp;Multithreading:</string>
     </property>
     <property name="buddy">
      <cstring>mtCombo</cstring>
     </property>
    </widget>
    <widget class="QComboBox" name="mtCombo">
     <property name="geometry">
      <rect>
       <x>130</x>
       <y>20</y>
       <width>221</width>
       <height>22</height>
      </rect>
     </property>
    </widget>
    <widget class="QCheckBox" name="cbSSE">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>60</y>
       <width>341</width>
       <height>17</height>
      </rect>
     </property>
     <property name="text">
      <string>Use &amp;SSE instruction set if available</string>
     </property>
    </widget>
    <widget class="QCheckBox" name="cbLowPriority">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>90</y>
       <width>341</width>
       <height>17</height>
      </rect>
     </property>
     <property name="text">
      <string>&amp;Yield resources to other programs</string>
     </property>
    </widget>
    <widget class="QCheckBox" name="cbUseCUDA">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>120</y>
       <width>341</width>
       <height>17</height>
      </rect>
     </property>
     <property name="text">
      <string>Use &amp;GPU computing if possible (NVIDIA CUDA)</string>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="tab">
    <attribute name="title">
     <string>&amp;Plot</string>
    </attribute>
    <widget class="QLabel" name="label_2">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>190</y>
       <width>111</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>&amp;Time Scale Units:</string>
     </property>
     <property name="buddy">
      <cstring>xsuCombo</cstring>
     </property>
    </widget>
    <widget class="QComboBox" name="xsuCombo">
     <property name="geometry">
      <rect>
       <x>140</x>
       <y>190</y>
       <width>221</width>
       <height>22</height>
      </rect>
     </property>
    </widget>
    <widget class="QGroupBox" name="cbDataFilter">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>20</y>
       <width>471</width>
       <height>71</height>
      </rect>
     </property>
     <property name="title">
      <string>Trajectory Smoothing</string>
     </property>
     <property name="checkable">
      <bool>true</bool>
     </property>
     <property name="checked">
      <bool>false</bool>
     </property>
     <widget class="QLabel" name="label_8">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>30</y>
        <width>361</width>
        <height>21</height>
       </rect>
      </property>
      <property name="text">
       <string>Central Moving Average sample &amp;width, in snapshots:</string>
      </property>
      <property name="buddy">
       <cstring>sbDataFilter</cstring>
      </property>
     </widget>
     <widget class="QSpinBox" name="sbDataFilter">
      <property name="geometry">
       <rect>
        <x>380</x>
        <y>30</y>
        <width>71</width>
        <height>22</height>
       </rect>
      </property>
      <property name="minimum">
       <number>1</number>
      </property>
      <property name="maximum">
       <number>999999</number>
      </property>
     </widget>
    </widget>
    <widget class="QCheckBox" name="cbMultisampleAA">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>110</y>
       <width>341</width>
       <height>17</height>
      </rect>
     </property>
     <property name="text">
      <string>Plot multi-sample &amp;anti-aliasing</string>
     </property>
    </widget>
    <widget class="Line" name="line_2">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>170</y>
       <width>471</width>
       <height>16</height>
      </rect>
     </property>
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
    </widget>
    <widget class="QCheckBox" name="cbPlotPolarAngles">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>140</y>
       <width>341</width>
       <height>17</height>
      </rect>
     </property>
     <property name="text">
      <string>Present angles in polar coordinates</string>
     </property>
    </widget>
    <widget class="QComboBox" name="langCombo">
     <property name="geometry">
      <rect>
       <x>350</x>
       <y>134</y>
       <width>141</width>
       <height>22</height>
      </rect>
     </property>
    </widget>
    <widget class="QLabel" name="label_9">
     <property name="geometry">
      <rect>
       <x>350</x>
       <y>110</y>
       <width>141</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>Plot Language:</string>
     </property>
     <property name="textFormat">
      <enum>Qt::AutoText</enum>
     </property>
     <property name="alignment">
      <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="calcTab">
    <attribute name="title">
     <string>&amp;Analysis</string>
    </attribute>
    <widget class="QGroupBox" name="groupBox">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>10</y>
       <width>491</width>
       <height>211</height>
      </rect>
     </property>
     <property name="title">
      <string>SAS and Occlusion</string>
     </property>
     <widget class="QLabel" name="label_5">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>30</y>
        <width>161</width>
        <height>21</height>
       </rect>
      </property>
      <property name="text">
       <string>&amp;Probe radius:</string>
      </property>
      <property name="buddy">
       <cstring>sasProbeRadius</cstring>
      </property>
     </widget>
     <widget class="QDoubleSpinBox" name="sasProbeRadius">
      <property name="geometry">
       <rect>
        <x>180</x>
        <y>30</y>
        <width>81</width>
        <height>22</height>
       </rect>
      </property>
      <property name="minimum">
       <double>0.100000000000000</double>
      </property>
      <property name="maximum">
       <double>50.000000000000000</double>
      </property>
      <property name="singleStep">
       <double>0.100000000000000</double>
      </property>
      <property name="value">
       <double>0.100000000000000</double>
      </property>
     </widget>
     <widget class="QLabel" name="label_6">
      <property name="geometry">
       <rect>
        <x>270</x>
        <y>30</y>
        <width>51</width>
        <height>21</height>
       </rect>
      </property>
      <property name="text">
       <string>A</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_7">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>60</y>
        <width>161</width>
        <height>21</height>
       </rect>
      </property>
      <property name="text">
       <string>&amp;Accuracy:</string>
      </property>
      <property name="buddy">
       <cstring>sasAccuracy</cstring>
      </property>
     </widget>
     <widget class="QComboBox" name="sasAccuracy">
      <property name="geometry">
       <rect>
        <x>180</x>
        <y>60</y>
        <width>81</width>
        <height>22</height>
       </rect>
      </property>
      <item>
       <property name="text">
        <string>Low</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Medium</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>High</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Very High</string>
       </property>
      </item>
     </widget>
     <widget class="QCheckBox" name="cbSasNoWater">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>100</y>
        <width>311</width>
        <height>21</height>
       </rect>
      </property>
      <property name="text">
       <string>E&amp;xclude water molecules from calculation</string>
      </property>
      <property name="checked">
       <bool>false</bool>
      </property>
     </widget>
    </widget>
   </widget>
   <widget class="QWidget" name="formatsTab">
    <attribute name="title">
     <string>&amp;Formats</string>
    </attribute>
    <widget class="QListWidget" name="formatList">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>20</y>
       <width>381</width>
       <height>192</height>
      </rect>
     </property>
     <property name="editTriggers">
      <set>QAbstractItemView::NoEditTriggers</set>
     </property>
    </widget>
    <widget class="QWidget" name="verticalLayoutWidget">
     <property name="geometry">
      <rect>
       <x>409</x>
       <y>20</y>
       <width>91</width>
       <height>191</height>
      </rect>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout">
      <item>
       <widget class="QPushButton" name="btnFormatAdd">
        <property name="text">
         <string>&amp;Add...</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="btnFormatEdit">
        <property name="text">
         <string>&amp;Edit...</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="btnFormatDelete">
        <property name="text">
         <string>&amp;Delete</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="btnFormatDefault">
        <property name="text">
         <string>&amp;Set Default</string>
        </property>
       </widget>
      </item>
      <item>
       <spacer name="verticalSpacer">
        <property name="orientation">
         <enum>Qt::Vertical</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>20</width>
          <height>40</height>
         </size>
        </property>
       </spacer>
      </item>
     </layout>
    </widget>
   </widget>
   <widget class="QWidget" name="colorsTab">
    <attribute name="title">
     <string>&amp;Colors</string>
    </attribute>
    <widget class="QListWidget" name="colorList">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>10</y>
       <width>241</width>
       <height>211</height>
      </rect>
     </property>
    </widget>
    <widget class="QLabel" name="lblColorName">
     <property name="geometry">
      <rect>
       <x>260</x>
       <y>10</y>
       <width>241</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="Line" name="line">
     <property name="geometry">
      <rect>
       <x>260</x>
       <y>30</y>
       <width>241</width>
       <height>16</height>
      </rect>
     </property>
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
    </widget>
    <widget class="QToolButton" name="colorChangeButton">
     <property name="geometry">
      <rect>
       <x>470</x>
       <y>50</y>
       <width>25</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>...</string>
     </property>
    </widget>
    <widget class="QLabel" name="lblColorValue">
     <property name="geometry">
      <rect>
       <x>260</x>
       <y>50</y>
       <width>201</width>
       <height>21</height>
      </rect>
     </property>
     <property name="autoFillBackground">
      <bool>true</bool>
     </property>
     <property name="frameShape">
      <enum>QFrame::StyledPanel</enum>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QLabel" name="lblText">
     <property name="geometry">
      <rect>
       <x>260</x>
       <y>80</y>
       <width>231</width>
       <height>41</height>
      </rect>
     </property>
     <property name="frameShape">
      <enum>QFrame::StyledPanel</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Plain</enum>
     </property>
     <property name="lineWidth">
      <number>1</number>
     </property>
     <property name="text">
      <string>Example text</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
    <widget class="QPushButton" name="fontButton">
     <property name="geometry">
      <rect>
       <x>260</x>
       <y>130</y>
       <width>75</width>
       <height>23</height>
      </rect>
     </property>
     <property name="text">
      <string>Fo&amp;nt...</string>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="viewerTab">
    <attribute name="title">
     <string>&amp;Viewer</string>
    </attribute>
    <widget class="QRadioButton" name="viewer0">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>20</y>
       <width>481</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>&amp;No PDB Viewer</string>
     </property>
    </widget>
    <widget class="QRadioButton" name="viewer1">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>50</y>
       <width>481</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>&amp;Internal PDB Viewer</string>
     </property>
     <property name="checkable">
      <bool>true</bool>
     </property>
    </widget>
    <widget class="QRadioButton" name="viewer2">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>80</y>
       <width>481</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>&amp;RasMol Viewer</string>
     </property>
     <property name="checkable">
      <bool>true</bool>
     </property>
    </widget>
    <widget class="QRadioButton" name="viewer3">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>140</y>
       <width>481</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>&amp;VMD Viewer</string>
     </property>
     <property name="checkable">
      <bool>true</bool>
     </property>
    </widget>
    <widget class="QLineEdit" name="strRasMol">
     <property name="enabled">
      <bool>false</bool>
     </property>
     <property name="geometry">
      <rect>
       <x>200</x>
       <y>110</y>
       <width>251</width>
       <height>20</height>
      </rect>
     </property>
    </widget>
    <widget class="QLabel" name="label_3">
     <property name="enabled">
      <bool>false</bool>
     </property>
     <property name="geometry">
      <rect>
       <x>40</x>
       <y>110</y>
       <width>161</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>Path to &amp;RasMol Executable:</string>
     </property>
     <property name="buddy">
      <cstring>strRasMol</cstring>
     </property>
    </widget>
    <widget class="QPushButton" name="browseRasMol">
     <property name="enabled">
      <bool>false</bool>
     </property>
     <property name="geometry">
      <rect>
       <x>450</x>
       <y>110</y>
       <width>22</width>
       <height>22</height>
      </rect>
     </property>
     <property name="text">
      <string>...</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_4">
     <property name="enabled">
      <bool>false</bool>
     </property>
     <property name="geometry">
      <rect>
       <x>40</x>
       <y>170</y>
       <width>161</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>Path to &amp;VMD Executable:</string>
     </property>
     <property name="buddy">
      <cstring>strVMD</cstring>
     </property>
    </widget>
    <widget class="QPushButton" name="browseVMD">
     <property name="enabled">
      <bool>false</bool>
     </property>
     <property name="geometry">
      <rect>
       <x>450</x>
       <y>170</y>
       <width>22</width>
       <height>22</height>
      </rect>
     </property>
     <property name="text">
      <string>...</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="strVMD">
     <property name="enabled">
      <bool>false</bool>
     </property>
     <property name="geometry">
      <rect>
       <x>200</x>
       <y>170</y>
       <width>251</width>
       <height>20</height>
      </rect>
     </property>
    </widget>
   </widget>
  </widget>
 </widget>
 <tabstops>
  <tabstop>tabWidget</tabstop>
  <tabstop>mtCombo</tabstop>
  <tabstop>cbSSE</tabstop>
  <tabstop>cbLowPriority</tabstop>
  <tabstop>cbUseCUDA</tabstop>
  <tabstop>cbDataFilter</tabstop>
  <tabstop>sbDataFilter</tabstop>
  <tabstop>cbMultisampleAA</tabstop>
  <tabstop>cbPlotPolarAngles</tabstop>
  <tabstop>langCombo</tabstop>
  <tabstop>xsuCombo</tabstop>
  <tabstop>sasProbeRadius</tabstop>
  <tabstop>sasAccuracy</tabstop>
  <tabstop>cbSasNoWater</tabstop>
  <tabstop>formatList</tabstop>
  <tabstop>btnFormatAdd</tabstop>
  <tabstop>btnFormatEdit</tabstop>
  <tabstop>btnFormatDelete</tabstop>
  <tabstop>btnFormatDefault</tabstop>
  <tabstop>colorList</tabstop>
  <tabstop>colorChangeButton</tabstop>
  <tabstop>fontButton</tabstop>
  <tabstop>viewer0</tabstop>
  <tabstop>viewer1</tabstop>
  <tabstop>viewer2</tabstop>
  <tabstop>strRasMol</tabstop>
  <tabstop>browseRasMol</tabstop>
  <tabstop>viewer3</tabstop>
  <tabstop>strVMD</tabstop>
  <tabstop>browseVMD</tabstop>
  <tabstop>buttonBox</tabstop>
 </tabstops>
 <resources/>
 <connections>
  <connection>
   <sender>viewer2</sender>
   <signal>toggled(bool)</signal>
   <receiver>label_3</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>132</x>
     <y>123</y>
    </hint>
    <hint type="destinationlabel">
     <x>135</x>
     <y>147</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>viewer2</sender>
   <signal>toggled(bool)</signal>
   <receiver>strRasMol</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>323</x>
     <y>124</y>
    </hint>
    <hint type="destinationlabel">
     <x>331</x>
     <y>153</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>viewer2</sender>
   <signal>toggled(bool)</signal>
   <receiver>browseRasMol</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>471</x>
     <y>127</y>
    </hint>
    <hint type="destinationlabel">
     <x>473</x>
     <y>149</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>viewer3</sender>
   <signal>toggled(bool)</signal>
   <receiver>label_4</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>79</x>
     <y>186</y>
    </hint>
    <hint type="destinationlabel">
     <x>83</x>
     <y>206</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>viewer3</sender>
   <signal>toggled(bool)</signal>
   <receiver>strVMD</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>302</x>
     <y>190</y>
    </hint>
    <hint type="destinationlabel">
     <x>305</x>
     <y>207</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>viewer3</sender>
   <signal>toggled(bool)</signal>
   <receiver>browseVMD</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>489</x>
     <y>186</y>
    </hint>
    <hint type="destinationlabel">
     <x>474</x>
     <y>209</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>preferencesDialog</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>318</x>
     <y>293</y>
    </hint>
    <hint type="destinationlabel">
     <x>391</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
